legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/validation/FunctionParametersParametersValidation.java

```diff
@@ -67,12 +70,20 @@ private static void validateParameterValues(RichIterable<Variable> externalParam
         }
     }
 
-    public static ValidationResult validate(Variable var, Object value)
+    public static ValidationResult validate(Variable var, List<ValidationContext> validationContext, Object value)
     {
         FunctionParameterTypeValidator validator = FunctionParameterTypeValidator.externalParameterTypeValidator(var._class);
         if (validator == null)
         {
-            return ValidationResult.errorValidationResult("Unknown external parameter type: " + var._class + ", valid external parameter types: " + FunctionParameterTypeValidator.getExternalParameterTypes().makeString("[", ", ", "]"));
+            ValidationContext enumContext = validationContext.stream().filter(v -> (v.getClass() == EnumValidationContext.class)).filter(e -> ((EnumValidationContext) e).varName.equals(var.name)).findFirst().orElse(null);
+            if (enumContext != null)
+            {
+                return (((EnumValidationContext)enumContext).validEnumValues.contains(value.toString())) ? ValidationResult.successValidationResult() : ValidationResult.errorValidationResult("Value " + value + " is not a valid enum value for " + var._class);
```

Comment:
Should we add the valid enum values to the error message, so users knows what is accepted?